IBPSA/Fluid/HeatPumps/ModularReversible/RefrigerantCycle/BaseClasses/PartialRefrigerantCycle.mo

```diff
@@ -0,0 +1,175 @@
+within IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.BaseClasses;
+partial model PartialRefrigerantCycle
+  "Partial model of refrigerant cycle"
+
+  parameter Modelica.Units.SI.HeatFlowRate QUse_flow_nominal
+    "Nominal heat flow rate at useful heat exchanger side"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Power PEle_nominal
+    "Nominal electrical power consumption"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Temperature TCon_nominal
+    "Nominal temperature at secondary condenser side"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Temperature TEva_nominal
+    "Nominal temperature at secondary evaporator side"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.TemperatureDifference dTCon_nominal
+    "Nominal temperature difference at secondary condenser side"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.TemperatureDifference dTEva_nominal
+    "Nominal temperature difference at secondary evaporator side"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mCon_flow_nominal
+    "Nominal mass flow rate in secondary condenser side"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mEva_flow_nominal
+    "Nominal mass flow rate in secondary evaporator side"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Real y_nominal(unit="1", min=0, max=1)=1
+    "Nominal relative compressor speed"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Real scaFac=QUse_flow_nominal/QUseNoSca_flow_nominal
+    "Scaling factor of heat pump" annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.HeatFlowRate QUseNoSca_flow_nominal
+    "Nominal heat flow rate at useful heat exchanger unscaled"
+    annotation (Dialog(group="Nominal condition"));
+  parameter String datSou=""
+    "Indicate where the data is coming from";
```

Comment:
This needs a better comment string that explain what is needed and/or how it is used. For example, in `Chillers.Examples.ModularReversible`, if the `RefrigerantCycleChillerCooling` is clicked, the gui shows
```modelica
 datSou = "Override to avoid warnings during simulation for CI"
   "Indicate where data is coming from"
```
Make sure all comments are clear, otherwise users are confused and/or send us questions.

Reply:
The setting in the example was a relict, at one point I wanted to check if the CI fails due to the assertion warning.
I updated the comment of `datSou` to avoid confusion.